### PR TITLE
Add missing Lambda enabled region 'eu-central-1' (Frankfurt)

### PR DESCRIPTION
**Pull request by erikerikson**

### CODE REVIEW

File: lib/Region.js [hunk 1]
```diff
@@ -70,6 +70,7 @@ module.exports = function(S) {
     static varsFilenameToRegionName(name) {
       if (name.includes('useast1'))      return 'us-east-1';
       if (name.includes('uswest2'))      return 'us-west-2';
+      if (name.includes('eucentral1'))   return 'eu-central-1';
       if (name.includes('euwest1'))      return 'eu-west-1';
       if (name.includes('eucentral1'))   return 'eu-central-1';
```
Comment (eahefnawy) [inline lib/Region.js]: We already added it! In line 75 :)

Comment (erikerikson) [inline lib/Region.js]: oh geez.  you did.  merge after fixing for the fail.  sorry!

Comment (erikerikson) [inline lib/Region.js]: and thanks for noticing!

**Comment by erikerikson** [comment]
already fixed

**Pull request closed without merging**